pkg/eval/builtin_fn_flow_test.go

```diff
@@ -30,7 +30,42 @@ func TestEach(t *testing.T) {
 	)
 }
 
-// TODO: test peach
+func TestPeach(t *testing.T) {
+	// Testing the `peach` builtin is a challenge since, by definition, the order of execution is
+	// undefined.
+	Test(t,
+		// Verify the output has the expected values when sorted.
+		That(`range 5 | peach [x]{ + 1 $x } | to-lines | order`).
+			Puts("1", "2", "3", "4", "5"),
```

Comment:
Now that #1309 is fixed the `to-lines` hack to get the numbers sortable is no longer needed.